src/createServer.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const usersServise = require('./services/users.js');
+const expensesServise = require('./services/expenses.js');
+const usersRouter = require('./routes/users.js');
+const expensesRouter = require('./routes/expenses.js');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  usersServise.getUsers();
+  expensesServise.getExpenses();
+
+  const app = express();
+
+  app.use('/users', express.json(), usersRouter);
```

Comment:
You are calling express.json() middleware for each route separately. It would be more efficient to call it once for the entire application, unless there are routes where you don't want to use it.